### ISSUE TITLE
Consider adding Seafile to the list of File Storage & Sync solutions

### ISSUE THREAD

**Issue by Zireael-N** [nylira/prism-break#1474]
Repositories: [server](https://github.com/haiwen/seafile), [desktop client](https://github.com/haiwen/seafile-client), [client for Android](https://github.com/haiwen/seadroid), [client for iOS](https://github.com/haiwen/seafile-iOS).

Wikipedia: [Seafile](https://en.wikipedia.org/wiki/Seafile).

> Seafile is a file hosting software system. Files are stored on a central server and can be synchronized with personal computers and mobile devices via the Seafile client. Files can also be accessed via the server's web interface.
>
> **Features**:
- Selective sync
- Differential sync to minimize bandwidth
- Client-side encryption
- User and group permission management
- Public link-sharing
- Version control
- Online file editing

**Comment by Sigmun** [comment]
Can an expert review this solution? I would like to set it up on my raspberry Pi...

**Comment by annonman** [comment]
I am a big fan of Seafile.  I'm not aware of any other FOSS project like this.

**Comment by nascheme** [comment]
Seafile is quite a good alternative to OneDrive, G-Drive, Dropbox, etc.  I think it should be listed as the code for it is open source.

**Comment by elouin** [comment]
Same here, and if there is a reason to not list it, i would love to know it.

**Comment by JoshuaRLi** [comment]
Vouching for seafile. Add it!

**Comment by macandchief** [comment]
I'm wondering: If there is Nextcloud under file storage and sync as a replacement for google drive, dropbox, etc. - why are Owncloud and Seafile missing?

**Comment by yegortimoshenko** [comment]
This is an "open core" product, which some people here are skeptical about. It's difficult to recommend free software version without recommending the one that's proprietary (which might have backdoors), and just from the way they split it they would have incentive to implement all new features in proprietary version first and foremost because that's the version they are paid to make.

**Comment by strugee** [comment]
Yeah, the file syncronization section is so overloaded my number one question is, what advantages does Seafile have that other solutions don't?

Also, @macandchief to answer your question (better late than never right?):

> why are Owncloud and Seafile missing?

ownCloud was replaced with Nextcloud because the latter is more featureful and less proprietary (see https://github.com/nylira/prism-break/issues/1551#issuecomment-272234839). As for Seafile, I don't know that there's a particular reason (although @yegortimoshenko mentioned that it's open core, which is a bummer although not a dealbreaker). It's mostly that no one's had time to look into it yet.

**Comment by strugee** [comment]
If no one knows of any really compelling advantages to Seafile I'd like to close this out.

**Comment by yegortimoshenko** [comment]
I forgot to substantiate my claim with a link: https://www.seafile.com/en/product/private_server/

**Comment by pbuyle** [comment]
I'm currently trying Seafile. The reason I choose Seafile over Nextcloud is because I prefer smaller tool that do one thing (files sync and cloud-hosting) than large kitchen-sink included solution (file sync and hosting, calendar, contacts, bookmarks, conference call, website building, collaborative office suite, etc.). It also seems to store data more efficiently on disk.

So the competitive advantage of Seafile would be being lightweight and narrow-focused.
